docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,41 @@
+[]
+= {WFM-RC-NameShort} Passport.js integration module
+Link to
+link:https://github.com/feedhenry-raincatcher/raincatcher-core/tree/master/cloud/passportauth[passportAuth] module.
+
+{WFM-RC-NameShort} authentication and authorisation module using link:http://passportjs.org[PassportJS].
+The Passport.js module is the default security module for {WFM-RC-NameShort}.
```

Comment:
The Passport.js module is the default authentication and authorisation module for {WFM-RC-NameShort}

"security" is far too vague.

Reply:
covered on previous sentence